anyway/models.py

```diff
@@ -47,13 +46,23 @@
     datefmt="%Y-%m-%d %H:%M:%S",
 )
 
-roles_users = Table(
-    "roles_users",
+users_to_roles2 = Table(
+    "users_to_roles2",
     Base.metadata,
-    Column("user_id", Integer(), ForeignKey("users.id")),
-    Column("role_id", Integer(), ForeignKey("roles.id")),
+    Column("id", Integer(), primary_key=True, autoincrement=True, nullable=False),
```

Comment:
Do we need the id field? (role-id and user-id are the actual key.)

Reply:
Replaced with role-id and user-id as the primary key.